source/_patterns/02-molecules/vue-widgets/src/vue-clock/vue-clock.vue

```diff
@@ -0,0 +1,210 @@
+<template>
+  <div class="container">
+    <h1>Dynamic classes based on Time</h1>
+    <div class="vue-clock">
+      <div
+        class="clock"
+        :class="[dynamicClasses]"
+      >
+        <div 
+          class="secondsAxis"
+          :style="[secondsPosition]"
+        >
+          <div class="secondsHand" />
+        </div>
+        <div
+          class="minutesAxis"
+          :style="[minutesPosition]"
+        >
+          <div class="minutesHand" />
+        </div>
+        <div
+          class="hoursAxis"
+          :style="[hoursPosition]"
+        >
+          <div class="hoursHand" />
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'VueClock',
+  data() {
+    return {
+      time: {
+        seconds: 0,
+        minutes: 0,
+        hours: 0,
+      },
+      secondsPosition: {
+        transform: 'rotate(0deg)',
```

Comment:
These should not be "styles" in the data. These should be values, the template should handle the inline style `transform`. Or a computed value.